### PR TITLE
KevS-YC-Translation-shared-responsibility-SAP-solution-for-OVHcloud

### PR DESCRIPTION
**Pull request by Kazarus4**
EN translation of RACI based on FR version. asked by @Y0Coss

### CODE REVIEW

File: pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+title: "Sharing responsibility for SAP on OVHcloud solutions"
+excerpt: "Shared responsibility between OVHcloud and the customer for using SAP on OVHcloud solutions"
+updated: 2023-12-08
+---
+
+## Objective
+
+The RACI below details the division of responsibilities between OVHcloud and the customer for SAP on OVHcloud solutions. This model can help the customer get the most out of SAP on OVHcloud solutions. It supplements the [RACI "Dedicated Servers"](/pages/account_and_service_management/responsibility_sharing/dedicated-servers) that applies to the SAP HANA on Bare Metal service, and the [RACI "VMware on OVHcloud"](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/responsibility-sharing) that applies to the SAP HANA on Private Cloud service. The acronym "SLES4SAP BYOL" is used for "SUSE Linux Enterprise Server for SAP Applications Bring Your Own License".
```
Comment (mlt-ovh) [inline pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md]: RACI "Dedicated Servers" =>"Dedicated Servers" RACI
RACI "VMware on OVHcloud" =>  "VMware on OVHcloud" RACI

File: pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+|C: Accessed during the process|
+|I: Is Notified of Process Results|
+
+### 1. Before subscription
+
+#### 1.1. Specify the service as needed
+
+| **SAP HANA on Bare Metal** | **SAP HANA on Private Cloud** | **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- | --- | --- |
+| X | X | Choose to subscribe to the [Dedicated Server](/pages/account_and_service_management/responsibility_sharing/dedicated-servers) service  or [VMware on OVHcloud](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/responsibility-sharing) depending on your needs, and follow the associated responsibility models | RA | CI |
```
Comment (mlt-ovh) [inline pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md]: Dedicated Server service or VMware on OVhcloud service
(more clear if we repeat the word "service I think")

File: pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+| X | X | Subscribe to SUSE for the OS license required to use the SLES4SAP BYOL image | RA |  |
+| X | X | You must own the license rights for the SAP software deployed on the infrastructure | RA |  |
```
Comment (mlt-ovh) [inline pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md]: Why "you must" ? => only "Own" no ?

File: pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+| X | X | Subscribe to the OVHcloud Object Storage offer if it is planned to use the OVHcloud Backint Agent for SAP HANA for backups | RA | CI |
+
+### 2. Service availability
+
+#### 2.1. Install the service
+
+| **SAP HANA on Bare Metal** | **SAP HANA on Private Cloud** | **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- | --- | --- |
+| X | X | Provision of the subscribed service and available features (OVHcloud Backint Agent for SAP HANA, OS image SLES4SAP BYOL) | I | RA |
```
Comment (mlt-ovh) [inline pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md]: Provision the (without "of") ?

File: pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+|  | X | Provision of the subscribed service and available features (Terraform modules for deploying SAP infrastructure as code, SAP HANA pre-installation) | I | RA |
```
Comment (mlt-ovh) [inline pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md]: Provision the (without "of") ?

File: pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+| X | X | Manage migration of SAP applications to OVHcloud | RA |  |
+
+#### 2.3. Installing the client IS
+
+| **SAP HANA on Bare Metal** | **SAP HANA on Private Cloud** | **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- | --- | --- |
+| X | X | Deploy the OS | RA |  |
+| X | X | Deploy the IT system as needed | RA |  |
+| X | X | Choose to configure SAP HANA backups with "OVHcloud Backint Agent for SAP HANA" | RA |  |
+|  | X | Choosing to launch the deployment of SAP infrastructure as code | RA |  |
```
Comment (mlt-ovh) [inline pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md]: Choose

File: pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+| X | X | Perform periodic restore tests of all SAP components, including the SAP HANA database | RA |  |
+| X | X | Maintain a disaster recovery and business continuity plan on the installed IS | RA |  |
+
+#### 3.2. Event management
+
+##### **3.2.1. Incidents**
+
+| **SAP HANA on Bare Metal** | **SAP HANA on Private Cloud** | **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- | --- | --- |
+| X | X | Analyzing and correcting proven development issues (bugs, faults, etc.) of the 'OVHcloud Backint Agent' | I | RA |
```
Comment (mlt-ovh) [inline pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md]: Analyze and correct

File: pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+| X | X | Handle incidents related to the SLES4SAP OS in BYOL mode and open tickets with the SUSE publisher if necessary | RA |  |
```
Comment (mlt-ovh) [inline pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md]: SUSE company (not publisher)

**Review by mlt-ovh — CHANGES_REQUESTED** [review]

File: pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md (hunk 5 above)
Comment (OVH-Team-Guides) [reply pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md]: @mlt-ovh , I suggest either "Provision of" or Provide the" (my preference is the 2nd choice)

File: pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md (hunk 4 above)
Comment (OVH-Team-Guides) [reply pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md]: @mlt-ovh , I suggest either "Provision of" or Provide the" (my preference is the 2nd choice)

File: pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md (hunk 5 above)
Comment (mlt-ovh) [reply pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md]: Ok for me

File: pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md (hunk 4 above)
Comment (mlt-ovh) [reply pages/hosted_private_cloud/sap_on_ovhcloud/sap_raci/guide.en-gb.md]: Ok for me

**Comment by OVH-Team-Guides** [comment]
@mlt-ovh , we applied your suggestions and did some minor edits. Can you please review again?

**Review by mlt-ovh — APPROVED** [review]

**Pull request merged**